modules/bvl_feedback/ajax/new_bvl_feedback.php

```diff
@@ -19,14 +19,20 @@
 //Creating a new array to pass the set values into the DB.
 $newThreadValues = [];
 
+// With formData, null is passed as a string
+foreach ($_POST as $key => $val) {
+    if ($_POST[$key] === 'null') {
```

Comment:
Seems like this would be a useful function in other places. Might be worth moving this function to the Utilities Class